src/Explorer/Panes/PublishNotebookPaneAdapter.tsx

```diff
@@ -104,11 +126,9 @@ export class PublishNotebookPaneAdapter implements ReactAdapter {
         this.imageSrc,
         this.content
       );
-      if (!response.data) {
-        throw new Error(`Received HTTP ${response.status} when publishing ${name} to gallery`);
+      if (response.data) {
+        NotificationConsoleUtils.logConsoleMessage(ConsoleDataType.Info, `Published ${name} to gallery`);
       }

```

Comment:
Should an error be displayed if there is no `response.data`?

Reply:
an error will be thrown from junoClient.publishNotebook() in case publish fails, the catch block will catch and display it.